osu.Game/Screens/Menu/ButtonSystem.cs

```diff
@@ -260,10 +263,16 @@ private bool goBack()
             {
                 case ButtonSystemState.TopLevel:
                     State = ButtonSystemState.Initial;
-                    sampleBack?.Play();
+
+                    // Samples are explicitly played here in response to user interaction and not when transitioning due to idle.
+                    StopSamplePlayback();
+                    sampleBackToLogo?.Play();
+                    sampleLogoSwoosh?.Play();
```

Comment:
Wondering if (only) the swoosh should also activate on the return-to-idle transition?

Reply:
I did try that but personally found it a bit weird, for example you'd be off alt-tabbed doing something else and randomly you'd get a swoosh in the background.

Reply:
Hmm I guess, when alt-tabbed I'd expect the game volume to already have dropped down from window being inactive though.

Reply:
Ah, personally I disable that volume ducking because I find the volume changes annoying 🙃. 

I'm just a bit reluctant to have sounds that randomly play after a delay, especially ones that aren't in response to user input is all... but if you think it's fine, I'll change the behaviour.